### ISSUE TITLE
Panel custom header aria-labelledby issue

### ISSUE THREAD

**Issue by ArpitaGhattuMicrosoft** [OfficeDev/office-ui-fabric-react#10003]
We are using Panel component and are seeing an accessibility issue(NVDA + chrome) where the screen reader is reading everything on the panel when opened. We are using a custom header and using the headerTextId as the id on the header element. But headerTextId is undefined and aria-labelledby is never set on the panel

## Codepen demonstrating the issue
https://codepen.io/anon/pen/vomXMq 

## Actual behavior:
Screen reader reads out everything without navigating on the panel.
Panel does not focus on header element on opening and screen reader is thrown off by it.

## Expected behavior:
Screen reader should just read the header and stop, until user navigates. 
Panel does not add aria-labelledby pointing towards the header element, and headerTextId is not undefined.

**Comment by marygans** [comment]
Hi @ArpitaGhattuMicrosoft! Thank you for submitting this issue to us. I agree that the expected behavior of the component should be to read the header text specified vs. the entire content in the panel. We will ensure this issue is addressed.

**Comment by ArpitaGhattuMicrosoft** [comment]
Thanks @marygans !

**Comment by marygans** [comment]
Hi @ArpitaGhattuMicrosoft! Thank you so much for you patience. I have been able to repro this issue with NVDA + Chrome but it turns out that the bug will actually need to be filed separately in their repo [here ](https://github.com/nvaccess/nvda/issues).

The NVDA screenreader in Edge and Narrator in both Chrome and Edge.

**Comment by rkalivarapu** [comment]
We are also facing this issue, there is no property for "Panel" like titleAriaId that was exposed for Dialog.

Can we have this property exposed for Panel?
